### ISSUE TITLE
WST Object?

### ISSUE THREAD

**Issue by cfpark00** [andrew-saydjari/DHC#2]
Do we want to start moving to Object Oriented?

The only reason I came to this is for the GPU version:
1. You want to minimize the number of sending data
2. You want to operate in batch-mode with pre-saved CUFFT.PlanMany, which really starts harnessing the power of GPUs.

-Core

**Comment by cfpark00** [comment]
Ok that was dumb! Julia doesn't have classes, I just found out. I'll do more reading

**Comment by dfink** [comment]
Yeah, I don’t know enough to have an opinion.  But I do suspect that a 256^3 FFT is still overhead dominated and not fully using the GPU.  Too small.  🙂
